dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'Magento_AdobeStockImageAdminUi/js/components/grid/column/overlay'
+], function ($, Overlay) {
+    'use strict';
+
+    describe('Magento_AdobeStockImageAdminUi/js/components/grid/column/overlay', function () {
+
+        var overlayObj;
+
+        beforeEach(function () {
+            overlayObj = new Overlay({
+                provider: 'providerName',
+                licensed: {}
+            });
+        });
+
+        describe('"initObservable" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('initObservable')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.initObservable;
+
+                expect(type).toEqual('function');
+            });
+
+            it('licensed will be defined', function () {
+                expect(overlayObj.licensed).toBeDefined();
+            });
+        });
+
+        describe('"updateLicensed" method', function () {
```

Comment:
While you have 'ajax' call success and failure test cases in this `describe` block, the `updateLicensed` method also does something special if the user is not logged in: it does not make an ajax call at all (it `return`s early). I think we should add a test for this.

Reply:
@konarshankar07 as mentioned I think a test checking that no ajax call is issued if the user is not authorized should be added.